### ISSUE TITLE
CMSG_CANCEL_QUEUED_SPELL no handler

### ISSUE THREAD

**Issue by Kitzunu** [WoWManiaUK/Blackwing-Lair#3176]
No handler

```cpp
    DEFINE_HANDLER(CMSG_CANCEL_QUEUED_SPELL,                              STATUS_UNHANDLED, PROCESS_INPLACE,      &WorldSession::Handle_NULL                     );
```

**Comment by Kitzunu** [comment]
Received not handled opcode [CMSG_CANCEL_QUEUED_SPELL 0x7B1C (31516)] from [Player: Combat GUID Full: 0x000000000000050f Type: Player Low: 1295, Account: 4491]

**Comment by Kitzunu** [comment]
does it give that error on wotlk then too? O.o

**Comment by Ovahlord** [comment]
Yes. It's a opcode that handles the cancellation of multiple pending spells. Up to MoP there was this spell batching that was delaying target spell casts up to 400ms so this opcode is to cancel all of them

**Comment by Rushor** [comment]
DEFINE_OPCODE_HANDLER(CMSG_CANCEL_MOUNT_AURA,                       STATUS_LOGGEDIN,  PROCESS_THREADUNSAFE, &WorldSession::HandleCancelMountAuraOpcode     );
